frappe_slack_connector/api/auth.py

```diff
@@ -8,7 +8,7 @@
 
 
 @frappe.whitelist()
-def connect_slack(user_email: str = None) -> None:
+def connect_slack(user_email: str = "") -> None:
```

Comment:
This would break, since in the next line there is a strict type check `if user_email is None`
Better fix would be to update the type annotation to: `(user_email: str | None = None)`